### ISSUE TITLE
check.assert.function(...) gives wrong message

### ISSUE THREAD

**Issue by myitcv** [philbooth/check-types.js#48]
As demonstrated by

```javascript
var check = require("check-types");

var x = 5;
var expectedMsg = "Invalid function";

try {
	check.assert.function(x);
} catch(e) {
	if(e.message !== expectedMsg) {
		console.log("Expected '"+expectedMsg+"'; got '" +e.message+"'");
	}
}
```

Looks like changing [`fn`](https://github.com/myitcv/check-types.js/blob/35c34956b0f74306837912ebc75287089b13ddfc/src/check-types.js#L28) to `function` will fix the issue?

**Comment by philbooth** [comment]
Oh good spot, thanks. There are no tests for the messages at all, I need to add some.

Were you planning to create a PR for the fix? If not I can sort something out over the weekend.

**Comment by myitcv** [comment]
@philbooth - perhaps #49 helps?

**Comment by philbooth** [comment]
Fixed by #49, published to npm as `3.3.1`.

**Comment by myitcv** [comment]
Thanks
